### PR TITLE
DAG-1637: Improve pull-request subcommand behavior

### PR DESCRIPTION
**Pull request by dbradf**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -188,11 +188,20 @@ for details.
 The `pull-request` subcommand will create pull requests across the base repo and all enabled 
 modules.  
 
-After local changes have committed in all repos, you can create the pull request from the base repo, and all
-enabled modules with changes would create a separate pull request. Each pull request would have comments that
-contain links for all other modules' pull requests.
+After local changes have committed in all repos, you can create the pull request from the base
```
Comment (zamj) [inline docs/usage.md:191]: [nit] have been committed

File: docs/usage.md [hunk 2]
```diff
@@ -188,11 +188,20 @@ for details.
 The `pull-request` subcommand will create pull requests across the base repo and all enabled 
 modules.  
 
-After local changes have committed in all repos, you can create the pull request from the base repo, and all
-enabled modules with changes would create a separate pull request. Each pull request would have comments that
-contain links for all other modules' pull requests.
+After local changes have committed in all repos, you can create the pull request from the base 
+repo, and all enabled modules with changes would create a separate pull request. Each pull 
+request would have comments that contain links for all other modules' pull requests.
```
Comment (zamj) [inline docs/usage.md:193]: [nit] "will create" "will have comments"

File: src/emm/emm_cli.py [hunk 3]
```diff
@@ -312,35 +319,34 @@ def git_commit(ctx: click.Context, commit_message: str) -> None:
...
+@click.option("--body", help="Body for the pull request")
 @click.pass_context
-def pull_request(ctx: click.Context) -> None:
+def pull_request(ctx: click.Context, title: Optional[str], body: Optional[str]) -> None:
     """
-    Create pull request for the changes in each module.
+    Create a Github pull request for changes in the base repository and any enabled modules.
 
-    Before use this command, you have to authenticate to github by 'gh auth login'.
+    NOTE: Before use this command, you have to authenticate to github by 'gh auth login'.
```
Comment (zamj) [inline src/emm/emm_cli.py:330]: [nit] Before using this command

**Review by zamj — APPROVED** [review]

File: docs/usage.md (hunk 1 above)
Comment (dbradf) [reply docs/usage.md]: done.

File: docs/usage.md (hunk 2 above)
Comment (dbradf) [reply docs/usage.md]: done.

File: src/emm/emm_cli.py (hunk 3 above)
Comment (dbradf) [reply src/emm/emm_cli.py]: done.

**Comment by dbradf** [comment]
evergreen merge

**Pull request merged**